lib/azure/armrest/storage/disk_service.rb

```diff
@@ -11,6 +11,115 @@ class DiskService < ResourceGroupBasedService
         def initialize(configuration, options = {})
           super(configuration, 'disks', 'Microsoft.Compute', options)
         end
+
+        # Get the raw blob information for a managed disk. This is similar to
+        # the StorageAccount#get_blob_raw method, but applies only to a managed
+        # disk, whereas that method applies only to an individual storage
+        # account.
+        #
+        # As with the Storage#get_blob_raw method, you should pass a :range,
+        # :start_byte, :end_byte or :length option. If you want the entire
+        # image you must pass the :entire_image option, though this is generally
+        # not recommended. Unlike the Storage#get_blob_raw method, this method
+        # does not support the :date parameter.
+        #
+        # You may also pass a :durationInSeconds parameter, which indicates
+        # how long the privately generated SAS token should last. This token
+        # is used internally by requests that are used to access the requested
+        # information. By default it lasts for 1 hour.
+        #
+        # Get the information you need using:
+        #
+        # * response.body    - blob data.
+        # * response.headers - blob metadata.
```

Comment:
What is the type of `response.headers` - is it `ResponseHeaders`? 
Same question for `response.body`.